src/Subscription.ts

```diff
@@ -126,4 +126,24 @@ export class UnsubscriptionError extends Error {
     super('unsubscriptoin error(s)');
     this.name = 'UnsubscriptionError';
   }
+}
+
+function callPrivateUnsubscribe(sub: Subscription): [boolean, any] {
```

Comment:
> These external functions are going to introduce closure when they're used

This `callPrivateUnsubscribe` don't capture a variables in outer environments. This would not be a closure which capture an environment. What would be a problem?